deploy/template/lib/systemd/system/docker-worker.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Taskcluster docker worker
+Requires=docker.service
+
+[Service]
+Type=notify
+ExecStart=/usr/local/bin/node /root/docker_worker/src/bin/worker.js --host test production
```

Comment:
will this be `--host packet` later?

Reply:
Yep, I need to work on the packet-net branch